ubuntu_image/__main__.py

```diff
@@ -20,6 +20,44 @@
 PROGRAM = 'ubuntu-image'
 
 
+class SimpleHelpFormatter(argparse.HelpFormatter):
+    def add_usage(self, usage, actions, groups, prefix=None):
+        # only show main usage when no subcommand is provided.
+        if prefix is None:
+            prefix = 'Usage: '
+        if len(actions) != 0:
+            usage = ('\n  {prog} COMMAND [OPTIONS]...'
+                     '\n  {prog} COMMAND --help').format(prog=PROGRAM)
+        else:
+            usage = ('{prog} ').format(prog=PROGRAM)
+
+        return super(SimpleHelpFormatter, self).add_usage(
+            usage, actions, groups, prefix)
+
+    def _format_action(self, action):
```

Comment:
Just a question for this particular method of the SimpleHelpFormatter. What purpose does it have? One of the visible changes is the removal of the subcommand choices line ({cmd1, cmd2}), but I can't see anything besides that - after removing it the help command seems to look similar as before. Could you briefly explain what it does? I might be missing something.

Reply:
Aha, you're right.
Thank you for your careful attention.
`({cmd1, cmd2})` looks ugly. Also, I didn't see such an analog in comparison with mainstream command line tool.
That's why I prefer this customized formatter. :)